stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/global/GlobalState.kt

```diff
@@ -59,7 +60,7 @@ public interface GlobalState {
             ]
         )
     )
-    public val initialized: StateFlow<Boolean>
+    public val initialized: Flow<Boolean>
```

Comment:
Why are we converting this to `Flow`? 🤔 I would rather keep it as a `StateFlow` and use `.stateIn()` in `GlobalMutablEstate`

Reply:
Can we revert this change?

Reply:
Yep, no problem